DataConnectors/ESETPROTECT/Connector_Syslog_ESETPROTECT.json

```diff
@@ -0,0 +1,154 @@
+{
+    "id": "ESETPROTECT",
+    "title": "ESET PROTECT",
+    "publisher": "ESET Netherlands",
```

Comment:
This Provider name is not used throughout. Should be changed to `ESET` to reflect that which is used all around.